.travis.yml

```diff
@@ -38,8 +38,9 @@ env:
   - VERSION="7.0" INCLUDE="test_module,second_module" ODOO_REPO="OCA/OCB" PG_LOGS_ENABLE="1" PG_NON_DURABILITY="1"  # ODOO_REPO usage example
   # - VERSION="6.1" INCLUDE="test_module,second_module"
   - LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="27" TRAVIS_PULL_REQUEST="false" # Use main pylint config file
-  - VERSION="master" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="84" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
-  - VERSION="7.0" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="73" TRAVIS_PULL_REQUEST="true"  # To check pylint_conf of PR's with old api
+  - VERSION="master" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="111" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
+  - VERSION="7.0" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="100" TRAVIS_PULL_REQUEST="true"  # To check pylint_conf of PR's with old api
+  - LINT_CHECK="1" TESTS="0" PYLINT_ODOO_JSLINTRC="DFTL_JSLINTRC" PYLINT_EXPECTED_ERRORS="111"
```

Comment:
Because I don't know whats the real path of the file that travis going to
take in account for the test. I don't know if it takes the .jslintrc from
pylint-odoo or I have to add the file into the same directory cfg with the
other config files.

El ago. 2, 2016 8:39 PM, "moylop260" <notifications@github.com> escribió:

> In .travis.yml
> <https://github.com/Vauxoo/maintainer-quality-tools/pull/131#discussion_r73268446>
> :
>
> > @@ -38,8 +38,9 @@ env:
> >    - VERSION="7.0" INCLUDE="test_module,second_module" ODOO_REPO="OCA/OCB" PG_LOGS_ENABLE="1" PG_NON_DURABILITY="1"  # ODOO_REPO usage example
> >    # - VERSION="6.1" INCLUDE="test_module,second_module"
> >    - LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="27" TRAVIS_PULL_REQUEST="false" # Use main pylint config file
> > -  - VERSION="master" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="84" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
> > -  - VERSION="7.0" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="73" TRAVIS_PULL_REQUEST="true"  # To check pylint_conf of PR's with old api
> > +  - VERSION="master" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="111" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
> > +  - VERSION="7.0" LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="100" TRAVIS_PULL_REQUEST="true"  # To check pylint_conf of PR's with old api
> > +  - LINT_CHECK="1" TESTS="0" PYLINT_ODOO_JSLINTRC="DFTL_JSLINTRC" PYLINT_EXPECTED_ERRORS="111"
>
> Why you are using the value DFTL_JSLINTRC ?
> You should use the full path of the file
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Vauxoo/maintainer-quality-tools/pull/131/files/8220900964d67c349038b193cd32b20fcbb43234#r73268446>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALMoyFzX29iSQ8EndyeKUDbQtojYZkdZks5qb_E-gaJpZM4Ja516>
> .
>